composer.json

```diff
@@ -11,6 +11,12 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "authors": [
+        {
+            "name": "Syde GmbH",
+            "homepage": "https://syde.com/",
+            "email": "hello@syde.com",
+            "role": "Company"
+        },
         {
             "name": "Christian Brückner",
             "email": "chris@chrico.info",
```

Comment:
That should be also updated to "Christian Leucht" und "c.leucht@syde.com" 😅

Reply:
Ok, I'll do it ASAP